iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -256,13 +256,15 @@ scf::ParallelOp collapseParallelLoops(ConversionPatternRewriter &rewriter,
   rewriter.setInsertionPointToStart(&newPLoopOp.getLoopBody().front());
   Value loopIv = *newPLoopOp.getInductionVars().begin();
   BlockAndValueMapping map;
+  edsc::ScopedContext scope(rewriter, loc);
```

Comment:
I am not sure we want to use EDSC in the main pipeline. EDSC is not a "blessed" approach in MLIR core. Its used in some places in Linalg, but I think there is no consensus on broader usage. So lets avoid it here.

Reply:
I see why edsc is being used. I guess with Linalg focused codegen paths and affine canonicalizations being important in propogating the static information, this is the best option. OK with leaving it in.

Reply:
Thanks, that makes the code simpler.